Sources/Extensions/Foundation/DateExtensions.swift

```diff
@@ -455,7 +455,9 @@ public extension Date {
     ///     let yesterday = date.yesterday // "Oct 2, 2018, 10:57:11"
     ///
     public var yesterday: Date {
-        return addingTimeInterval(-86400.0)
+        let cal = Calendar.current
```

Comment:
Please do this in a different PR

Reply:
@AlexeiGitH please remove these changes and fix the conflicts so that we can finally merge this in 😅